Remotion/Web/Core/Res/Themes/NovaGray/HTML/SingleView.css

```diff
@@ -20,37 +20,26 @@ div.singleView > div.wrapper
   right: 0;
   width: auto;
   height: auto;
-}
 
-div.singleViewView
-{
-  height: 80%;
-  width: inherit;
+  display: grid;
```

Comment:
I was thinking of splitting this up (screenreader/ie and gridmode). The two changes are not related.